assets/src/shared/hooks/use-operations-supervisors.tsx

```diff
@@ -0,0 +1,242 @@
+import { useManageSupervisors } from '~/api/use-manage-supervisors';
+import { useInvalidateSupervisors } from '~/api/use-get-supervisors';
+import { checkValidResultSuccess } from '~/shared/lib/check-valid-result-success';
+import { notifyError } from '~/shared/lib/notify-error';
+import { toastManager } from '~/shared/lib/toastManager';
+
+export const useOperationsSupervisors = () => {
+  const manageSupervisors = useManageSupervisors();
+  const invalidateSupervisors = useInvalidateSupervisors();
+
+  /// --- Servers ----
```

Comment:
It seems to me that this comment and the following ones are unnecessary, ideally if the code can be understood without comments, in this case everything is already clear

What do you think?

Reply:
fixed - ec5f137ebcc2ff71ab39ddc9be779a34669f66e4 ;)